### PR TITLE
STORM-2237: Nimbus reports bad supervisor heartbeat - unknown version or thrift exception

### PR DESCRIPTION
**Pull request by knusbaum**

### CODE REVIEW

**Comment by knusbaum** [comment]
I'll have a version up for 2.x as well.

File: storm-core/src/ui/public/index.html [hunk 1]
```diff
@@ -169,6 +175,12 @@ <h2 id="cluster-resources-header">Cluster Resources</h2>
               ]
             });
             $('#supervisor-summary [data-toggle="tooltip"]').tooltip();
+            //$('#bad-supervisor-summary [data-toggle="tooltip"]').tooltip();
```
Comment (d2r) [inline storm-core/src/ui/public/index.html]: Does this need to be here?

**Comment by d2r** [comment]
Seems good to me +1

**Comment by harshach** [comment]
+1

**Comment by HeartSaVioR** [comment]
+1
@knusbaum Could you provide a patch for master branch? Thanks in advance.

**Comment by erikdw** [comment]
@knusbaum : can you please put in some details about what is wrong, how it's fixed, etc.?

**Comment by HeartSaVioR** [comment]
Any updates here?

**Comment by HeartSaVioR** [comment]
@knusbaum Kindly reminder

**Comment by HeartSaVioR** [comment]
@knusbaum Kindly reminder for requesting forward port.

**Pull request closed without merging**